connectors/sources/servicenow.py

```diff
@@ -324,9 +324,21 @@ async def filter_services(self, configured_service):
                 ]
                 async for table_data in self.get_data(batched_apis=batched_apis):
                     for mapping in table_data:  # pyright: ignore
-                        if mapping["label"] in invalid_services:
-                            servicenow_mapping[mapping["label"]] = mapping["name"]
-                            invalid_services.remove(mapping["label"])
+                        name = mapping.get("name")
+                        if not name:
+                            msg = "Entry in sys_db_object without sysparm_field 'name' was found. This is a non-issue if no invalid services are flagged."
+                            self._logger.debug(msg)
```

Comment:
I'm not a servicenow expert but can we at least provide some "id" of a mapping that is missing name or label? Otherwise it's a bit hard to debug from the user perspective